project/core/templates/static/less/landing.css

```
@@ -0,0 +1,548 @@
+    border-radius: 1em;
+  }
+  .ripple:active {
+    background-color: #5918d1;
+    background-size: 100%;
+    transition: background 0s;
+    border-radius: 1em;
+  }
+  
+  /* Button style */
+  button {
+    border: none;
+    border-radius: 1em;
+    padding: 12px 18px;
+    font-size: 16px;
+    cursor: pointer;
+    color: white;
+    background-color: #6526de;
+    /* box-shadow: 0 0 4px #999; */
+    outline: none;
+  }
+  
+  .landing-msg {
+    position: relative;
+    top: 50%;
+    width: 24em;
+    margin: 0 auto;
+    /* border-right: 2px solid rgba(136, 16, 184, 0.75); */
+    font-size: 180%;
+    text-align: center;
+    white-space: nowrap;
+    overflow:hidden;
+    /* transform: translateY(-50%); */
+  }
+  
+  /* Animation */
+  .anim-typewriter {
+    animation: typewriter 4s steps(44) 1s 1 normal both,
+      blinkTextCursor 500ms steps(44) infinite normal;
+  }
+  @keyframes typewriter {
+    from {
+      width: 0;
+    }
+    to {
+      width: 24em;
+    }
+  }
+  @keyframes blinkTextCursor {
+    from {
+      border-right-color: rgba(166, 23, 202, 0.685);
+    }
+    to {
+      border-right-color: transparent;
+    }
+  }
+  
+  .landing {
+      background-image: url("../img/bg.jpg");
+      height: 90vh;
+  }
+  
+  .btn-container {
+      padding: 2em;
+  }
+  
+  .msg-1 {
+      margin: 2em,2em,2em,2em;
+      text-align: center;
+  }
+  
+  .title {
+      font-weight: 600;
+      font-size: 40px;
+      padding: 1em;
+      text-align: center;
+  }
+  
+  .subtitle {
+      font-weight: 400;
+      font-size: 1.5em;
+      margin-left: 2em;
+      margin-right: 2em;
+  
+  }
+  
+  .video-text {
+      font-size: 1.5em;
+      padding-bottom: 2em;
+      font-weight: 300;
+  }
+  
+  .video {
+      margin-top: 2em;
+      text-align: center;    
+  } 
+  
+  .body-link {
+      color: #4712aa;
+  }
+  
+  .body-link a:visited {
+      color: #0f0f0f;
+  }
+  
+  .body-link a:hover {
+      color: #4917a7;
+  }
+  
+  .why {
+      margin-top: 3em;
+      background-color: aliceblue;
+  }
+  
+  .text {
+      padding-top: 0.75em;
+      margin-left: 2em;
+      margin-right: 2em;
+      font-size: 1.15em;
+  }
+  
+  .why-table {
+      margin-left: auto;
+      margin-right: auto;
+      max-width: 80%;
+  }
+  
+  .tr-why {
+      padding-top: 1em;
+      padding-bottom: 0.5em;
+  }
+  
+  .tr-why:nth-child(2) {
+      text-align: right;
+  }
+  
+  .material-icons {
+      font-size: 10em;
+      text-align: center !important;
+      margin-left: auto;
+      margin-right: auto;
+  }
+  
+  .center {
+      text-align: center;
+  }
+  
+  footer {
+      background-color: #333333;
+      padding: 2em;
+  }
+  
+  th {
+      color: #eeeeee;
+  }
+  
+  .footer {
+      color: #eeeeee;
+      text-align: right;
+      /* padding: 2em; */
+      margin-left: 70%;
+  }
+  .footer a{
+      color: #9e9e9e;
+  }
+  
+  .footer td {
+      padding-left: 10px;
+      padding-bottom: 2px;
+  }
```

Add a new line at the end of files :-)
```suggestion
  }
  
```